environment.sample

```diff
@@ -30,9 +30,9 @@ ODOO_PASSWORD=
 #SENTRY_DSN=
 
 # Number of approvals to have the proposal marked as "Approved"
-#APPROVALS_REQUIRED=2
+APPROVALS_REQUIRED=2
 # Number of days before the proposal can be marked as "Approved"
-#MIN_PR_AGE=5
+MIN_PR_AGE=5
```

Comment:
having a sample file that contains default value commented out is good practice. So I'd keep that.